lib/dpp/DriveStateRepository.js

```diff
@@ -49,6 +49,15 @@ class DriveStateRepository {
     const transaction = this.getDBTransaction('identity');
 
     await this.identityRepository.store(identity, transaction);
+
+    const storeMapPromises = identity.getPublicKeys()
```

Comment:
We already have `storePublicKeyIdentityId`

Reply:
Why don't you want to do it in another PR?

Reply:
We don't need this logic. It's already implemented in `storePublicKeyIdentityId` method.